### PR TITLE
Next.js and Node.js docs

### LINKED ISSUE #7234: Add NodeJS tracing doc
Structure should follow [this](https://www.notion.so/runhighlight/Traces-Plans-832cca9c1f6346af92bd5c826a9fc623?pvs=4).Need to make sure quickstart guides are linked correctly in the main app as well.

### LINKED ISSUE #7235: Add NextJS Tracing doc
Structure should follow [this](https://www.notion.so/runhighlight/Traces-Plans-832cca9c1f6346af92bd5c826a9fc623?pvs=4).Need to make sure quickstart guides are linked correctly in the main app as well.

### PR DESCRIPTION
**Pull request by deltaepsilon**
## Next.js

Closes #7235 

image

image

image

## Node.js

Closes #7234 

image

### CODE REVIEW

File: highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	subtitle:
+		'Learn how to set up highlight.io tracing for your Next.js application.',
+	logoUrl: siteUrl('/images/quickstart/nextjs.svg'),
+	entries: [
+		{
+			title: 'Install the Next.js SDK',
+			content: 'Install the Next.js SDK using your package manger.',
+			code: [
+				{
+					text: `npm install --save @highlight-run/next`,
```
Comment (Vadman97) [inline highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx:16]: we have a helper for this snippet
https://github.com/highlight/highlight/blob/main/highlight.io/components/QuickstartContent/backend/js/shared-snippets.tsx#L4-L36

File: docs-content/getting-started/6_backend-tracing/2_node-js/3_nextjs.md
Comment (Vadman97) [inline docs-content/getting-started/6_backend-tracing/2_node-js/3_nextjs.md:1]: can we also update https://highlight-landing-lc1f10y2b-highlight-run.vercel.app/docs/sdk/nextjs to include the new tracing APIs?

File: highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx
Comment (Vadman97) [inline highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx:1]: we should link `/nextjs` from the overview page https://highlight-landing-git-ce-docs-highlight-run.vercel.app/docs/getting-started/backend-tracing/node-js/overview

**Review by Vadman97 — APPROVED** [review]
🎉

File: highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx (hunk 1 above)
Comment (deltaepsilon) [reply highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx:16]: How do you feel about removing references to `yarn` and `pnpm` site-wide?

We removed them on the Next.js docs, and I kinda like it? Anyone using `yarn`, `pnpm` or `bun` knows the correct commands, so it tightens up our docs.

image

File: docs-content/getting-started/6_backend-tracing/2_node-js/3_nextjs.md
Comment (deltaepsilon) [reply docs-content/getting-started/6_backend-tracing/2_node-js/3_nextjs.md:1]: What do you think of this?

image

File: highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx
Comment (deltaepsilon) [reply highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx:1]: image

**Pull request merged**